lib/WUI/sntp/sntp_opts.h

```diff
@@ -78,7 +78,7 @@
  * \#define SNTP_SERVER_ADDRESS "pool.ntp.org"
  */
 #if !defined SNTP_SERVER_DNS || defined __DOXYGEN__
-    #define SNTP_SERVER_DNS 0
+    #define SNTP_SERVER_DNS 1
```

Comment:
These options probably should go to `include/buddy/lwipopts.h` (see the top of this file, it's included from there and this is just creating a default).

Reply:
Yes, I will have a look.

Reply:
Changed that.